packages/common/src/types/api/comments.ts

```diff
@@ -7,7 +7,9 @@ export type ApiCreateComment = {
 
 export type ApiGetComments = {
     status: 'ok';
-    results: ApiCommentsResults;
```

Comment:
:+1: good thing you removed `Record` from the API, we shouldn't use it.

Reply:
nice!